packages/entities/entities-shared/src/composables/useFetcher.ts

```diff
@@ -54,8 +56,13 @@ export default function useFetcher(
       const dataKey = (dataKeyName && dataKeyName.replace(/[^\w-_]/gi, ''))
       let tableData
 
-      if (data[dataKey] && Array.isArray(data[dataKey])) {
-        tableData = data[dataKey]
+
+      if (data[dataKey]) {
+        if (Array.isArray(data[dataKey])) {
+          tableData = data[dataKey]
+        } else {
+          tableData = [data[dataKey]]
+        }
```

Comment:
Do you think this is better?
```suggestion
        tableData = Array.isArray(data[dataKey]) ? data[dataKey] : [data[dataKey]]
```